docs/content/latest/benchmark/tpcc.md

```diff
@@ -0,0 +1,101 @@
+---
+title: TPCC
+linkTitle: TPCC
+description: TPCC
+image: /images/section_icons/architecture/concepts.png
+headcontent: Benchmark YugabyteDB using TPCC.
+menu:
+  latest:
+    identifier: benchmark-tpcc
+    parent: benchmark
+    weight: 740
+aliases:
+  - /benchmark/tpcc/
+showAsideToc: True
+isTocNested: True
+---
+
+
+## Step 1. Install Java and Ant.
+
+## Step 2. Clone the TPCC repository.
+
+You can do this by running the following commands.
+
+```sh
+cd $HOME
+git clone https://github.com/yugabyte/tpcc.git
+cd tpcc
+```
+
+## Step 3. Compile the code
+
+You can compile the code using the following commands.
+
+```sh
+ant bootstrap
+ant resolve
+ant build
+```
+
+## Step 4. Start your database
+Start the database using steps mentioned here: https://docs.yugabyte.com/latest/quick-start/explore-ysql/.
+
+## Step 5. Configure connection properties
+Set the following connection configurations in yugabyteSQL/db.properties:
```

Comment:
What is yugabyteSQL/db.properties? Is this a new file that needs to be created?

Since we already have the config files committed, it'll be easier to just ask users to modify ip address in `config/workload_all` if needed.

Reply:
Changed the statement.